frontend/apps/desktop/tests/onboarding-from-scratch.e2e.ts

```diff
@@ -0,0 +1,66 @@
+import {test} from '../test/fixtures'
+
+import {expect} from '@playwright/test'
+
+test('Onboarding from scratch', async ({onboardingPage}) => {
+  let {appWindow} = onboardingPage.appData
+  await test.step('Welcome Screen', async () => {
+    let startBtn = await appWindow.locator('#btn-new-account')
+    await expect(startBtn).toBeVisible()
+    await startBtn.click()
+  })
+
+  await test.step('Secret Recovery phrase', async () => {
+    let elPhrase = await appWindow.locator('#recovery-phrase-words')
+    let phraseTxt = await elPhrase.textContent()
+    let elNextBtn = await appWindow.locator('#btn-next')
+    let elCheck1 = await appWindow.locator('#check1')
+    let elCheck2 = await appWindow.locator('#check2')
+    await expect(phraseTxt).toBeTruthy()
+    await expect(phraseTxt?.split(' ').length).toEqual(12)
+    await expect(elNextBtn).toBeDisabled()
+    await elCheck1.check()
+    await elCheck2.check()
+    expect(elNextBtn).not.toBeDisabled()
+    await elNextBtn.click()
+  })
+
+  await test.step('Profile Data', async () => {
+    let elAlias = appWindow.locator('#alias')
+    let elBio = appWindow.locator('#bio')
+    let elNextBtn = await appWindow.locator('#btn-next')
+
+    await elAlias.fill('testAlias')
+    await elBio.fill('test bio')
+
+    await elNextBtn.click()
+  })
+
+  await test.step('Wallet', async () => {
+    // TODO: uncomment wallet accept flow when backend is fixed
```

Comment:
We should resolve this, usually its a matter of 3~5 seconds so the daemon is ready. But could not test it locally due to heap memory issues

Reply:
UPDATE: this is resolved. we increment the time to 10_000 just in case